.typos.toml

```diff
@@ -10,6 +10,7 @@ extend-exclude = [
   "tasks/prettier_conformance/prettier",
   "crates/oxc_parser/src/lexer/mod.rs",
   "crates/oxc_linter/fixtures",
+  "crates/oxc_linter/src/rules/jsx_a11y/img_redundant_alt.rs",
```

Comment:
https://github.com/oxc-project/oxc/pull/1571/files#diff-d06ab4ebdb38671afdf28c5d9b48cfc2e5a80255b1d73eaa7324f7df9338b576R231

some test case in img_redundant_alt rule has intentional typo in file, and I could not find any way to disable typo check within per one line, so I ignore `crates/oxc_linter/src/rules/jsx_a11y/img_redundant_alt.rs`...